bots/elasticsearch/bot.py

```diff
@@ -150,6 +256,18 @@ def setup(self):
         logger.debug('Waiting for yellow status')
         self.es_client.cluster.health(wait_for_status='yellow')
 
+        logger.info('Putting Elasticsearch settings')
+        self.es_client.indices.close(
+            index=settings.ELASTICSEARCH_INDEX,
+        )
+        self.es_client.indices.put_settings(
```

Comment:
Ohh, *that's* why I got random "index closed" errors while messing with mappings. 👍